invenio_vocabularies/contrib/subjects/facets.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Northwestern University.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Subjects services."""
+
+
+class SubjectsLabels:
+    """Fetching of subjects labels for facets."""
+
+    def __call__(self, ids):
+        """Return the mapping when evaluated.
+
+        For subjects, the ids received are actually human-readable and coming
```

Comment:
Is this the case for every subject scheme items? I understand the scheme itself is ok (e.g. MeSH is a human readable) but the items... i.e.

- MeSH: https://raw.githubusercontent.com/galterlibrary/invenio-subjects-mesh/master/invenio_subjects_mesh/vocabularies/subjects_mesh.yaml
- FOS: https://github.com/inveniosoftware/cookiecutter-invenio-rdm/pull/189/files

They both have a URL in the `id` field. I'm wondering then what's the frontend sending as id if it's human readable 🤔

Reply:
Ah maybe `ids` is what is tripping up here. These `ids` are actually either the vocabulary `scheme` (top-level) or `subject` (nested level). Not the vocabulary `id`. That's what the comment was about. `ids` as an argument was kept to be *consistent* with the interface of the `Labels` category of classes (e.g. [`VocabularyLabels`](https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/services/facets.py#L39)).

What about `label_keys` as a better argument name for both this class and `VocabularyLabels` ?

Reply:
Ah I see, thanks. Maybe clarifying it in the docstring is enough (like putting there almost your answer from above).

Reply:
ok I just reworded the existing comment.